### DISCUSSION TITLE
Training with Lora - Launch then... failed why please ?

### DISCUSSION THREAD

**Discussion by Maranpani** [bmaltais/kohya_ss#199] [Q&A]
I follow all th step that Olivio tell (in this video :https://youtu.be/9MT1n97ITaE ), I click on train model. SOmething begin then it's failed. I'am not good into informatic and all this. IS there a genius that could help me to resolve my issue to be able to train the AI and create my first model please ?

DAta\test\model"================================================================================
CUDA SETUP: Loading binary C:\Users\Utilisateur\Documents\Kohya\kohya_ss\venv\lib\site-packages\bitsandbytes\libbitsandbytes_cuda116.dll...
use 8-bit Adam optimizer
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 1700
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 850
  num epochs / epoch数: 1
  batch size per device / バッチサイズ: 2
  total train batch size (with parallel & distributed & accumulation) / 総バッチサイズ（並列学習、勾配合計含む）: 2
  gradient ccumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 850
steps:   0%|                                                                                   | 0/850 [00:00<?, ?it/s]epoch 1/1
Traceback (most recent call last):
  File "C:\Users\Utilisateur\Documents\Kohya\kohya_ss\train_db.py", line 355, in <module>
    train(args)
  File "C:\Users\Utilisateur\Documents\Kohya\kohya_ss\train_db.py", line 241, in train
    if args.noise_offset:
AttributeError: 'Namespace' object has no attribute 'noise_offset'
steps:   0%|                                                                                   | 0/850 [00:38<?, ?it/s]
Traceback (most recent call last):
  File "C:\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Utilisateur\Documents\Kohya\kohya_ss\venv\Scripts\accelerate.exe\__main__.py", line 7, in <module>
  File "C:\Users\Utilisateur\Documents\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\accelerate_cli.py", line 45, in main
    args.func(args)
  File "C:\Users\Utilisateur\Documents\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py", line 1104, in launch_command
    simple_launcher(args)
  File "C:\Users\Utilisateur\Documents\Kohya\kohya_ss\venv\lib\site-packages\accelerate\commands\launch.py", line 567, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['C:\\Users\\Utilisateur\\Documents\\Kohya\\kohya_ss\\venv\\Scripts\\python.exe', 'train_db.py', '--enable_bucket', '--pretrained_model_name_or_path=//UTILISATEUR-PC/Users/Utilisateur/stable-diffusion-webui/models/Stable-diffusion/realisticVisionV13_v13.safetensors', '--train_data_dir=C:\\Users\\Utilisateur\\Documents\\Lora TRaining DAta\\test\\image', '--resolution=512,512', '--output_dir=C:\\Users\\Utilisateur\\Documents\\Lora TRaining DAta\\test\\model', '--logging_dir=C:\\Users\\Utilisateur\\Documents\\Lora TRaining DAta\\test\\log', '--save_model_as=safetensors', '--output_name=last', '--learning_rate=1e-5', '--lr_scheduler=cosine', '--lr_warmup_steps=85', '--train_batch_size=2', '--max_train_steps=850', '--save_every_n_epochs=1', '--mixed_precision=fp16', '--save_precision=fp16', '--seed=1234', '--cache_latents', '--bucket_reso_steps=64', '--mem_eff_attn', '--gradient_checkpointing', '--xformers', '--use_8bit_adam', '--bucket_no_upscale']' returned non-zero exit status 1.

**Comment by quasiblob** [comment]
This was already answered in another thread:
https://github.com/bmaltais/kohya_ss/discussions/193

A solution is to replace one python script:
https://github.com/bmaltais/kohya_ss/issues/192#issuecomment-1435526398

This fixed this at least for me, although I have other issues with this GUI.
